hspec-core/src/Test/Hspec/Core/Spec/Monad.hs

```diff
@@ -79,12 +101,19 @@ mapSpecItem _ = mapSpecItem_
 mapSpecItem_ :: (Item a -> Item b) -> SpecWith a -> SpecWith b
 mapSpecItem_ = mapSpecForest . bimapForest id
 
+-- | Sets the `Params` on the items to be generated by `SpecWith`.
```

Comment:
```suggestion
-- | Modifies the `Params` on the items to be generated by `SpecWith`.
```

This is a bit pedantic, feel free to ignore - obviously `const newParams` for `f` here is the "setter" version.